msteams-platform/m365-apps/overview.md

```diff
@@ -52,17 +52,17 @@ Build your app with the latest [Teams app manifest](../resources/schema/manifest
 
 With an aim toward simplifying and streamlining the Microsoft 365 developer ecosystem, we're continuing to expand the Teams app manifest into other areas of Microsoft 365 with the following.
 
-### Office Add-ins (preview)
+### Microsoft 365 Add-ins (preview)
 
-You can now define and deploy Office Add-ins in the [developer preview version](../resources/schema/manifest-schema-dev-preview.md) of the Microsoft Teams app manifest. Currently, this preview is limited to Outlook Add-ins running on subscription Office for Windows.
+You can now define and deploy Microsoft 365 Add-ins in the [developer preview version](../resources/schema/manifest-schema-dev-preview.md) of the Microsoft Teams app manifest. Currently, this preview is limited to Outlook Add-ins running on subscription Microsoft 365 for Windows.
```

Comment:
The extensibility model is still called "Office Add-ins" (its just the end-user Office app that's now rebranded as M365 app)